src/components/Torrents.vue

```diff
@@ -40,7 +40,7 @@
           :title="$t('force_start')"
           :disabled="!hasSelected"
         >
-          <v-icon>mdi-play-speed</v-icon>
+          <v-icon>mdi-step-forward</v-icon>
```

Comment:
`skip-forward` better

Reply:
It is not. That is explicitly related to music, specifically tapes.